src/Nethermind/Nethermind.TxPool/ITxPoolConfig.cs

```diff
@@ -28,5 +28,8 @@ public interface ITxPoolConfig : IConfig
         
         [ConfigItem(DefaultValue = "5")]
         int PeerNotificationThreshold { get; set; }
+        
+        [ConfigItem(DefaultValue = "2048", Description = "Max number of transactions held in mempool (more transactions in mempool mean more memory used")]
+        public int Size { get; set; }
```

Comment:
Does it implement it in interface (public?), if yes init to default size. If we want to implement it in class remove public?

Reply:
oh wow, this whole new interface thing is a massive nonsense